### ISSUE TITLE
Check training metrics are not getting too bad between versions

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#853]
We're currently checking per-date, maybe we should do between versions too? Or is it redundant?

**Comment by Lothiraldan** [comment]
I don't think we run the training with version N-1 once we deployed version N so we should be covered by the check per-date for regressions.

But it could be helpful to see the difference, if any, between version N-1 and version N. But any change on the accuracy for example could also be due to the new data we trained the model against or the new data we used to test the model.
